client/src/hooks/useForgotPasswordForm.jsx

```diff
@@ -0,0 +1,103 @@
+import { useDisclosure, useToast } from '@chakra-ui/react';
+import { useRef, useState } from 'react';
+import useEmailValidation from 'hooks/useEmailValidation.jsx';
+import { userForgotPassword } from 'utils/api/users.js';
+import { sendEmail } from 'utils/api/system.js';
+
+const useForgotPasswordForm = () => {
+  const emailRef = useRef('');
+  const [emailIsValidated, setEmailIsValidated] = useState(true);
+  const [humanIsValidated, setHumanIsValidated] = useState(true);
+  const [buttonIsDisabled, setButtonIsDisabled] = useState(true);
+  const toast = useToast();
+  const [isLoading, setIsLoading] = useState(false);
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const { isAnEmail } = useEmailValidation();
+
+  const forgotPasswordHandleSubmit = async (e) => {
+    e.preventDefault();
+    setIsLoading(true);
+
+    const email = emailRef.current.value;
+
+    try {
+      const promise = await userForgotPassword({
+        email: email,
+      });
+      const url = window.location.href.substr(0, window.location.href.lastIndexOf('/'));
+
+      try {
+        await sendEmail({
+          to: [email],
+          subject: 'ERP Password Reset',
+          emailBody: `This is the ERP Team07,\n\nPlease reset your password using the following link: ${url}/reset/${promise.data}\n\nRegards.`,
+        });
+        toast({
+          title: 'Email',
+          description: 'Email Sent',
+          status: 'success',
+          duration: 9000,
+          isClosable: true,
+        });
+      } catch (err) {
+        toast({
+          title: 'Error',
+          description: 'An error occured while sending the email',
+          status: 'error',
+          duration: 9000,
+          isClosable: true,
+        });
+      }
+    } catch (err) {
+      // User Not Found, but do not tell it to the user.
+    }
+
+    toast({
```

Comment:
this should be removed because this messaged is displayed even if we were not able to send an email

Reply:
The reason for this is to avoid "bad" people spamming email and see if the client/employee email is inside the database or no.